config/routes.rb

```diff
@@ -3,12 +3,16 @@
   devise_for :users,
              :controllers => { :registrations => "my_devise/registrations" }
 
-  resources :courses, only: [:index, :show]
+  resources :courses
   resources :course_registrations, only: [:destroy]
+  resources :instructors, only: [:index, :create, :edit, :update, :destroy]
+  resources :venues, only: [:index, :new, :create, :edit, :update]
 
   resources :students, only: [:new, :create] do
     resources :registration_holders, only: [:new, :index, :create]
   end
 
   resources :students, only: [:index, :show, :destroy]
+
+  get 'dojo', to: 'dojos#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.